### PR TITLE
bump nodejs dependencies based on npm audit

### PR DESCRIPTION
**Pull request by ohadlevy**
this pr ignores jquery 3 update and compression-webpack-plugin as they are handled in other prs and require potential code chagnge

### CODE REVIEW

**Comment by ohadlevy** [comment]
katello test failures seem unrelated.

File: package.json [hunk 1]
```diff
@@ -61,7 +61,7 @@
     "babel-polyfill": "^6.26.0",
     "brace": "^0.10.0",
     "datatables.net": "~1.10.12",
-    "datatables.net-bs": "~1.10.12",
+    "datatables.net-bs": "^2.1.1",
```
Comment (tbrisker) [inline package.json]: iirc, "datatables.net-bs" is the bootstrap styling for datatables. Are you sure bumping this version without changing the matching "datatables.net" version works?

Comment (ohadlevy) [reply package.json]: I did confirm it works (nothing seems to fail and the page seemed to render correctly).
I do hope we will migrate away from datatables sooner than later anyway..

Comment (ekohl) [reply package.json]: This specific version is marked as deprecated on NPM. Are you sure this is the correct version? https://www.npmjs.com/package/datatables.net-bs lists 1.10.16 as the current version.

File: package.json [hunk 2]
```diff
@@ -39,16 +39,16 @@
     "jest-cli": "^20.0.0",
     "jsdom": "^9.5.0",
     "lodash-webpack-plugin": "^0.11.4",
-    "node-sass": "^4.5.0",
+    "node-sass": "^4.9.0",
```
Comment (ekohl) [inline package.json]: This is a rather complicated package to build. Could we submit this as a separate PR so we can get all the other changes in first?

**Review by ekohl — COMMENTED** [review]
I've opened https://github.com/theforeman/foreman-packaging/pull/2673. This is the first package we handle where something depends on `@x/y` where everything else just had `x` as names. Let's see if our build process handles this now.

**Comment by ohadlevy** [comment]
updated, removed datatables.net-bs and node-sass from the pr.

**Comment by ohadlevy** [comment]
also, would you like a redmine issue (one or more?) for these?

**Review by ekohl — APPROVED** [review]
Personally I don't think a redmine issue is that important in this case but it could be considered good practice to always do so.

**Review by mmoll — APPROVED** [review]

**Comment by mmoll** [comment]
Test failures are unrelated.

I didn't actually test this on a foreman instance and leave it to the RPM people to actually merge.

**Comment by ohadlevy** [comment]
@ekohl bump?

**Comment by ekohl** [comment]
@ohadlevy I already gave an approval
